hummingbot/client/command/list_command.py

```diff
@@ -65,16 +67,35 @@ def list(self,  # type: HummingbotApplication
 
         elif obj == "trades":
             lines = []
-            if self.strategy is None:
-                self._notify("No strategy available, cannot show past trades.")
+            #To access the trades from Markets Recorder you need the file path and strategy name
+            if in_memory_config_map.get("strategy_file_path").value is None or \
+                    in_memory_config_map.get("strategy").value is None:
+                self._notify("Kindly Configure the bot first")
             else:
-                if len(self.strategy.trades) > 0:
-                    df = Trade.to_pandas(self.strategy.trades)
-                    df_lines = str(df).split("\n")
-                    lines.extend(["", "  Past trades:"] +
-                                 ["    " + line for line in df_lines])
+                markets_recorder = MarketsRecorder(
```

Comment:
You should be able to use `self.markets_recorder` here instead of building your own.

Reply:
Already did that, that does not work unless a strategy is running